flink-1.17-connector-bigquery/flink-connector-bigquery/src/main/java/com/google/cloud/flink/bigquery/table/BigQueryDynamicTableFactory.java

```diff
@@ -102,11 +113,32 @@ public DynamicTableSource createDynamicTableSource(Context context) {
             config = config.withTestingServices(testingServices);
         }
 
+        // Create an unbounded source.
```

Comment:
this comment is a little bit misleading. the boundedness of the source we create here is dependent on the config options (ie, won't always be unbounded)

Reply:
Fixed!